buildscript.txt

```diff
@@ -3,6 +3,6 @@ chat
 --docker-repos=gcr.io/overleaf-ops
 --env-add=
 --env-pass-through=
---node-version=10.23.1
+--node-version=12.21.0
 --public-repo=False
---script-version=3.4.0
+--script-version=3.7.0
```

Comment:
Why did we need to update this?

Reply:
Presumably you ran the buildscript as the method to upgrade the node version. Just want to confirm though.

Reply:
yes. that's what happened